### PR TITLE
Replace static icon references with symbol API

### PR DESCRIPTION
**Pull request by NotMyFault**
This PR replaces a few static icon references with already used symbols through the symbol API. I plan to create a follow-up PR to address the leftover ones by adding the corresponding symbols.

noJob.groovy - the arrow shape change is marginal
<details>
<summary>Before</summary>

Screenshot 2022-09-03 at 22 28 29

</details>

<details>
<summary>After</summary>

Screenshot 2022-09-03 at 21 10 42

</details>

headerContent.jelly - Same symbol like on the profile page header
<details>
<summary>Before</summary>

Screenshot 2022-09-03 at 22 29 09

</details>

<details>
<summary>After</summary>

Screenshot 2022-09-03 at 21 11 24
</details>

<l:helpIcon>
<details>
<summary>Before</summary>

Screenshot 2022-09-03 at 22 30 59

</details>

<details>
<summary>After</summary>

https://github.com/jenkinsci/jenkins/blob/master/war/src/main/resources/images/symbols/help-circle.svg

</details>

### Proposed changelog entries

- Minor changes to empty state arrow icon and header user profile icon

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  - Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")` if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content-Security-Policy directives (see [documentation on jenkins.io](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7052"></a>

### CODE REVIEW

File: core/src/main/resources/hudson/util/JenkinsReloadFailed/index.groovy
Comment (basil) [inline core/src/main/resources/hudson/util/JenkinsReloadFailed/index.groovy:12]: I don't feel strongly about this, but perhaps the existing `close-circle`, `close`, or `indeterminate` icons would work for this. Or we could add `alert`, `alert-circle`, `bug`, `nuclear`, or `warning` from upstream Ionicons. Can be tested with

```java
diff --git a/core/src/main/java/jenkins/model/Jenkins.java b/core/src/main/java/jenkins/model/Jenkins.java
index 66b7a9e6ab..bbdd39a359 100644
--- a/core/src/main/java/jenkins/model/Jenkins.java
+++ b/core/src/main/java/jenkins/model/Jenkins.java
@@ -4338,9 +4338,9 @@ public class Jenkins extends AbstractCIBase implements DirectlyModifiableTopLeve
                 try (ACLContext ctx = ACL.as2(ACL.SYSTEM2)) {
                     reload();
                     getLifecycle().onReady();
+                    new JenkinsReloadFailed(new Throwable()).publish(servletContext, root);
```

and reloading Jenkins.

Comment (NotMyFault) [reply core/src/main/resources/hudson/util/JenkinsReloadFailed/index.groovy:12]: To be fair, this is a left over from the header icon cleanup, I didn't want to create a dedicated PR form when I noticed it.

I can do so if you want.

Comment (basil) [reply core/src/main/resources/hudson/util/JenkinsReloadFailed/index.groovy:12]: I am not familiar with the header icon cleanup. If this is consistent with some previously accepted PR that you can link to, that is fine with me.

Comment (NotMyFault) [reply core/src/main/resources/hudson/util/JenkinsReloadFailed/index.groovy:12]: This took place across a series of PRs modernizing different components, for example https://github.com/jenkinsci/jenkins/pull/5925/files#diff-58a21354a8d8b1ac4db72b5a141485cd14f888acd5f32e5148db76779c753bafL33
The goal was to remove icons next to headers, as it is the current state with the majority of pages.

Comment (basil) [reply core/src/main/resources/hudson/util/JenkinsReloadFailed/index.groovy:12]: OK, would have been nice if this was done consistently throughout the repository in a single commit, but as long as this is consistent with previous changes I am fine with it.

**Comment by basil** [comment]
> @NotMyFault requested a review from [basil](https://github.com/basil) 8 minutes ago

I have no idea how to review this change. If you want me to approve it, you just need to convince me that you've tested the change sufficiently. Your review has no "Testing Done" section, so while I generally trust your work I have no way of verifying that.

**Comment by NotMyFault** [comment]
> I have no idea how to review this change.

No worries, I updated the PR description to include a bit more context, but I wouldn't mind if you pass on your review to other UX SIG members, if you prefer that.

**Comment by timja** [comment]
Before / After screenshots of every component changed here would make this easier rather than just the result

**Comment by NotMyFault** [comment]
> Before / After screenshots of every component changed here would make this easier rather than just the result

There we go 👍🏻

**Review by timja — APPROVED** [review]
Tested locally, looks good and thanks for the screenshots

**Comment by timja** [comment]
@NotMyFault I think this is worth adding to the changelog as a minor item, i.e. the user profile icon is changing

**Review by janfaracik — APPROVED** [review]

**Comment by NotMyFault** [comment]
This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Review by basil — APPROVED** [review]

**Comment by daniel-beck** [comment]
`skip-changelog`?

**Comment by timja** [comment]
Discussed in PR, could go either way

File: core/src/main/resources/lib/layout/helpIcon.jelly
Comment (daniel-beck) [inline core/src/main/resources/lib/layout/helpIcon.jelly:22]: Caused [SECURITY-2886](https://www.jenkins.io/security/advisory/2022-09-21/#SECURITY-2886).

**Pull request merged**
